sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/security/validation/CustomObjectMapperAdvice.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022- SDA SE Open Industry Solutions (https://www.sda.se)
+ *
+ * Use of this source code is governed by an MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+package org.sdase.commons.spring.boot.web.security.validation;
+
+import javax.annotation.PostConstruct;
+import org.sdase.commons.spring.boot.web.jackson.SdaObjectMapperConfiguration;
+import org.sdase.commons.spring.boot.web.security.exception.InsecureConfigurationException;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.stereotype.Component;
+
+/**
+ * Checks that custom error mappers are registered by the SdaObjectMapperConfiguration. The check is
+ * indirectly performed by checking that the required bean is in the spring context. This class
+ * checks for the risks identified in the security guide as:
+ *
+ * <ul>
+ *   <li>"Risiko: Erkennung von vertraulichen Komponenten ... Entfernen von applikations-bezogenen
```

Comment:
Could we add an English version of this risk?

Reply:
I put it on my list.